cmd/config_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"os"
+	"testing"
+)
+
+func TestNewCommandWithTomlConfigFile(t *testing.T) {
+	cmd, err := invokeProxyCommand([]string{"--config-file", "../testdata/cloud-sql-proxy.config.test-toml.toml"})
```

Comment:
Let's make all these tests table tests. We'll need to add a setup/cleanup function:

``` golang
# table test data looks like this
var tcs := []struct{
  desc string
  args []string
  setup: func() func()
}{
  desc: "some description here",
  args: []string{ /*...*/ },
  setup: func() { 
    // t.Setenv and t.Cleanup might be another good option
    os.Setenv("CSQL_PROXY_DEBUG", false) 
    return func() { os.Unsetenv("CSQL_PROXY_DEBUG") }
  },
}

for _, tc := range tcs {
  t.Run(tc.desc, func(t *testing.T) {
    cleanup := setup()
    defer cleanup()

    // use tc.args and assert as necessary
  }
}
```

Reply:
Also let's move the testdata directory under cmd, so we don't have to go up a directory to reach it (`../testdata` vs `testdata`).

Reply:
done!